common/pom.xml

```diff
@@ -87,5 +87,16 @@
                 </includes>
             </resource>
         </resources>
+        <testResources>
```

Comment:
Why add this？

Reply:
There were so many branches of code, I import ApprovalTests for test, Approvaltests will generate an approval.txt in the same level directory of the test class. approval.txt is needed when executing the test case. If not, the test case will fail. The purpose of adding these codes is that MVN compiletest can put approval.txt in the test class directory, so that it will not fail in CI, But I still fail after I do this, I don't know why, I'm still looking for it.

---

源码中分支很多, 所以我引入了 ApprovalTests 去测试, ApprovalTests 会生成一个 approval.txt 在测试类的同级目录, 在执行测试用例的时候需要 approval.txt, 如果没有这个文件那么测试用例将会失败, 我加上这些代码的目的是希望再 mvn compileTest 能把 approval.txt 打进 test-class 目录中, 以至于在CI时候不会失败, 但是我这样做了之后还是会失败, 我不太清楚为什么, 我还在找原因.

Reply:
如果分支过多，不方便补充用例的话，就先补充我们使用的场景相关的测试。